packages/pwa/webpack.config.js

```diff
@@ -4,6 +4,94 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-const config = require('pwa-kit-react-sdk/webpack/config')
+const webpackConfig = require('pwa-kit-react-sdk/webpack/config')
+const {PwaKitConfigPlugin} = require('pwa-kit-react-sdk/webpack/plugins')
```

Comment:
I believe I shared my opinion on this before, but I'll share it again. We can hash it out later 🤣 

But my preference would be to have a dedicated schema file for the app in a known location, which would allow us to not mess with app webpack config and load it up in the plugin directly. Here are my arguments for why I like that solution:

1. The intentions of the app's webpack config was to simply show how someone can import it modify it, and export it. It's not really a place where we are going to put code that would be universal to all starter apps.
2. We already make an assumption where the config file (the definition) is,..  is it really that much worse to also make a default to where the schema lives (we can add an option to configure it).
3. Changing the structure, adding or removing properties to a schema is probably something that might happen often, if I was asked to make a change to the schema I wouldn't thing that the webpack config would be the first place I'd look to make a change. But looking at a file called `pwa-kit-config.schema` that sits right next to `pwa-kit-config.json` would be more of an obvious choice. 

Downside, is that it would make mocking for testing the plugin a little more cumbersome, but not impossible. 


**UPDATE** 
After looking at what the `customSchemas` is actually structured like, its making more sense on why you made the decisions you did. From the looks of it the pwa doesn't have a single schema but a list of schema's that it has to obey. Maybe we should talk this out a little.